BraveRewardsUI/Tipping/TippingViewController.swift

```diff
@@ -63,6 +72,73 @@ class TippingViewController: UIViewController, UIViewControllerTransitioningDele
     tippingView.optionSelectionView.options = [1.0, 5.0, 10.0].map {
       TippingOption.batAmount(BATValue($0), dollarValue: state.ledger.dollarStringForBATAmount($0) ?? "")
     }
+    
+    loadPublisherBanner()
+  }
+  
+  // MARK: - Networking
+  private func loadPublisherBanner() {
+    state.ledger.publisherBanner(forId: self.publisherInfo.id) { [weak self] banner in
+      guard let self = self, let banner = banner else { return }
+      
+      self.tippingView.overviewView.titleLabel.text = banner.title.isEmpty ? Strings.TippingOverviewTitle : banner.title
+      self.tippingView.overviewView.bodyLabel.text = banner.desc.isEmpty ? Strings.TippingOverviewBody : banner.desc
+      
+      self.downloadImage(url: banner.background, { image in
+        self.tippingView.overviewView.headerView.image = image
+      })
+      
+      self.downloadImage(url: self.publisherInfo.faviconUrl, { image in
+        self.tippingView.overviewView.faviconImageView.image = image
+      })
+      
+      self.tippingView.overviewView.socialStackView.arrangedSubviews.forEach({ $0.removeFromSuperview() })
+      
+      if !banner.social.isEmpty {
+        banner.social.keys.forEach({
+          //When testing with "3zsistemi.si", the values come back empty, but we still have valid keys..
+          //It seems we don't care about the values as long as the key exists, that user has this media..
+          if let mediaType = BravePublisherSocialMedia(rawValue: $0) {
+            let icons: [BravePublisherSocialMedia: String] = [.youtube: "pub-youtube",
```

Comment:
Instead of this dictionary we could create a helper in `BravePublisherSocialMedia` enum that will return correct icon, something like

````
// in BravePublisherSocialMedia:
var iconName: String {
  switch self {
    case .youtube: return "pub-youtube"
    ...
  }
}
```

and then use it at line 107 as `icon.iconName`

Reply:
Fixed.